packages/flutter/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  parse_server_sdk: ^3.1.2
+  parse_server_sdk: ^3.1.3
```

Comment:
I think this also needs to be reverted, otherwise we'd have to release a new flutter version?

Reply:
No, there is no need Returned to the previous version